Gemfile.lock

```diff
@@ -2,71 +2,131 @@ GEM
   remote: http://rubygems.org/
```

Comment:
From what I saw in testing, deprecation of features seems well handled, and we test locally with Jekyll before push anyway. I have no strong opinion either way, so I'm happy to drop the Gemfile.lock in this PR if need be.